pokete_data/poketes.py

```diff
@@ -1333,6 +1333,100 @@
    ((((""",
             "esc": None}]
     },
+    "bablbam": {
+        "name": "Bablbam",
+        "hp": 20,
+        "atc": 2,
+        "defense": 1,
+        "attacks": ["tackle", "mind_blow"],
+        "pool": [],
+        "miss_chance": 0,
+        "desc": "Dead or alive entity with slow movement and mental ability.",
+        "lose_xp": 3,
+        "rarity": 1,
+        "types": ["undead"],
+        "evolve_poke": "",
+        "evolve_lvl": 0,
+        "initiative": 1,
+        "ico": [{
+            "txt": r"""   (.)~
+  -)_(-U
+ / ; \ """,
+            "esc": None}]
+    },
+    "kakraholt": {
+        "name": "Kakraholt",
+        "hp": 25,
+        "atc": 6,
+        "defense": 3,
+        "attacks": ["shock", "charging", "mega_arch", "tackle"],
+        "pool": [],
+        "miss_chance": 0,
+        "desc": "A terrestrial crab full of accumulated electrical energy.",
+        "lose_xp": 4,
+        "rarity": 0.5,
+        "types": ["electro", "normal"],
+        "evolve_poke": "",
+        "evolve_lvl": 0,
+        "initiative": 5,
+        "ico": [{
+            "txt": r"""     _
+   +O O+
+   /(+)\->>
+  /\|^|/\  """,
```

Comment:
```suggestion
    O O
   /( )\
  /\|^|/\  """,
```
Can be removed, since it's overlayed anyways.

Reply:
Thanks, redundant characters from layers are gone